Modified: README

stpk007 opened the pull request:
**Zmiana nazwy i rozszerzenia pliku (lepsza widoczność), zmiana struktury pliku** (dodanie nowego tłumacza, opis w języku polskim na pierwszym miejscu), dodanie informacji o witrynie _Weblate_ -- proszę o komentarz i ewentualne poprawki do pliku.

Commit:
Modified: README

Commit:
Rename README to README.md

Commit:
Update README.md

Commit:
Update README.md

Commit:
Rename README.md to README-pl-PL

Commit:
Update README-pl-PL

Commit:
new file:   pl-PL/.directory
	modified:   pl-PL/README-pl-PL

Commit:
Delete .directory

mateunho commented:
1. Trzymamy się konwencji zgodnie z pozostałymi zespołami i pozostawiamy nazwę `README` oraz format
2. Zmiany struktury - czekam na uzasadnienie
3. Commity ze zmianami idącymi `upstream` do repo Alioth powinny zaczynać się prefiksem `pl-PL: ` (porównaj z innymi zespołami)
4. Liczba commitów jest zbyt duża. Poza tym tytuły powinny obrazować zmiany, a wyglądają chaotycznie i nie mówią wiele o zawartości commita.

> ad 4. https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

Commit:
Added README-pl-PL to the main directory

W ten sposób nasz projekt jest bardziej widoczny, w końcu jest projekt dotyczący tłumaczenia na język polski!

Commit:
Delete README-pl-PL

Commit:
Rename README-pl-PL to README-pl-PL.md

ghost commented:
On Sat, November 26, 2016 15:04, Mateusz Kacprzak wrote:
> 1. Trzymamy się konwencji zgodnie z pozostałymi zespołami i pozostawiamy nazwę
> `README` oraz format
Ok Jesli taka jest konwencja, to cofam propozycję zmiany nazwy.
> 2. Zmiany struktury - czekam na uzasadnienie

Ponieważ jesteśmy zespołem tłumaczy z języka angielskiego na język polski, to
naszym preferowanym językiem powinien być nasz język ojczysty -- tak mi się
przynajmniej wydaje. Zdaję sobie sprawe, że plik README mogą też czytać osoby
znające tylko język angielski, ale na odrobinę "nacjonalizmu" (w dobrym
znaczeniu tego słowa) można chyba sobie pozwolić?

Pozostałe kwestie techniczne pozostawiam nietknięte, jeśli takie są wymogi.

A tak w ogóle, to powracam do Weblate, bo te "eksperymenty" po pierwsze,
trochę mnie już zmęczyły (nadal nie do końca chyba opanowałem GitHub), a po
drugie, to oderwałem się trochę od mojej zasadniczej aktywności, tzn. od
tłumaczenia sensu stricto.

Pozdrawiam  - Stanisław.






> 3. Commity ze zmianami idącymi `upstream` do repo Alioth powinny zaczynać się
> prefiksem `pl-PL: ` (porównaj z innymi zespołami)
> 4. Liczba commitów jest zbyt duża. Poza tym tytuły powinny obrazować zmiany, a
> wyglądają chaotycznie i nie mówią wiele o zawartości commita.
>
>> ad 4. https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/debian-handbook-pl/pl-PL/pull/93#issuecomment-263065094

Commit:
-restored the previous form and name of the file,
        -added the essential information about the project.

        The information has been updated and made more credible
        (please, read carefully all the text, without a bias): pl-PL/README

	deleted: pl-PL/README-pl-PL.md (the explanation unnecessary)
